.buildkite/pipeline.yml

```diff
@@ -239,6 +239,20 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":computer: no lim ARS baroclinic wave (ρe) equilmoist with EDMFX"
+        command:
+          - julia --color=yes --project=examples examples/hybrid/driver.jl --job_id sphere_baroclinic_wave_rhoe_equilmoist_edmfx --initial_condition MoistBaroclinicWaveWithEDMF --moist equil --precip_model 0M --turbconv edmfx --apply_limiter false --dt 450secs --t_end 6days --dt_save_to_disk 2days --reference_job_id sphere_baroclinic_wave_rhoe_equilmoist
+        artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist_edmfx/*"
+        agents:
+          slurm_mem: 20GB
+
+      - label: ":computer: EDMFX Bomex on a sphere"
+        command:
+          - julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_bomex_sphere --initial_condition Bomex --moist equil --turbconv edmfx --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --apply_limiter false --z_elem 60 --z_stretch false --z_max 3e3 --dt 60secs --t_end 1hours --dt_save_to_disk 5mins --reference_job_id edmf_bomex
```

Comment:
Should we also add a "consistency test" fir EDMFX?

Reply:
I don't think we'll need a consistency test if we design our code to avoid dependency issues. Moving values from the "aux state" into local (broadcastable) functions will be a big part of that.